site/smart-accounts/accounts/light-account.md

```diff
@@ -12,41 +12,49 @@ head:
       content: Follow this guide to use Light Account with Account Kit, a vertically integrated stack for building apps that support ERC-4337.
   - - meta
     - name: twitter:title
-      content: Light Account Smart Contract
+      content: Light Account
   - - meta
     - name: twitter:description
       content: Follow this guide to use Light Account with Account Kit, a vertically integrated stack for building apps that support ERC-4337.
 ---
 
-# Light Account Smart Contract
+# Light Account
 
-Light Account is a secure, gas-optimized, [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) smart account implementation.
+**Secure, optimized, and extendable**
```

Comment:
is it extendable?  it's not eip-6900 compatible so what do we mean by this?

Reply:
meaning forward-compatible with modular account via future upgrade

Reply:
ah so maybe the keyword was upgradeable?

Reply:
yeah, but upgradeable kind overlapped with the general upgradeable contracts concept, but that's not really what I tried to entail. More of high level umbrella term to imply the overall picture. No strong pref